docs/other/payload-validation.mdx

```diff
@@ -0,0 +1,124 @@
+---
+id: payload_validation
+title: Payload validation with AMF
+---
+
+AMF can be used to validate incoming payloads to an API already deployed and running.
+For example, consider a RAML API that declares:
+
+- a `User` schema with `firstname`, `lastname` and `age` properties
+- a `/users` endpoint with an HTTP POST method that requires a `User` object as body parameter
+
+```yaml
+#%RAML 1.0
+title: API with Types
+types:
+  User:
+    type: object
+    properties:
+      firstname: string
+      lastname:  string
+      age:       number
+/users:
+  post:
+    body:
+      application/json:
+        type: User
+```
+
+How can you confirm that a given payload is a valid `User`? You can do that with AMF payload validation.
+To do that, you first need to go through certain steps:
+
+1. Get the AMF payload shape
+2. Create a `PayloadValidator`
+3. Validate payloads
+
+
+## 1. Get the AMF payload model
```

Comment:
## Get the AMF payload model

I don't think we need--or have been using--numbers in the headings. (Please correct me if I'm wrong... 😜 )